package.json

```diff
@@ -62,6 +62,7 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.19.0",
+    "eslint-plugin-react-hooks": "^2.3.0",
```

Comment:
Is this not part of `eslint-config-airbnb`?

Reply:
I will check I think this was part of your original PR or from other PRs since this was opened, but will confirm and make the changes required

Reply:
Just checked and its a peer dependency of airbnb - so needs to be included here its the same with some of the other above dependencies 😄

Reply:
Not addressed due to the above